packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html

```diff
@@ -3,12 +3,12 @@
     <div class="topic-teaser-container bg-light">
       <div class="row pt-huge-r">
 
-        <div class="topic-teaser-content col-12 col-lg-8">
+        <div class="topic-teaser-content">
           <h2 class="topic-teaser-title font-curve-large mb-large">
             <span class="bold">Lorem ipsum</span>
             <span class="light">sit amet, consetetur sadipscing elitr</span>
           </h2>
-          <ul class="link-list font-curve-regular mb-large">
+          <ul class="link-list font-curve-small mb-large">
```

Comment:
The link list font-size should be 20px on desktop:

https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components?node-id=174%3A200